lib/dotcom_web/components/planned_disruptions.ex

```diff
@@ -139,7 +142,7 @@ defmodule DotcomWeb.Components.PlannedDisruptions do
     if Timex.after?(service_date_datetime, service_date_today) do
       service_date_datetime |> Timex.format!("%a, %b %-d", :strftime)
```

Comment:
We probably need to do something about this too. For example, `Hoy – Thu, Sep 11: Shuttles` should probably be `Hoy – Jue, Sep 11: Shuttles`

Reply:
Yeah, I've localized a few things for the trip planner, mostly in https://github.com/mbta/dotcom/blob/main/lib/dotcom_web/components/trip_planner/helpers.ex... can use custom format a la [this example](https://github.com/mbta/dotcom/blob/main/lib/dotcom_web/components/trip_planner/helpers.ex#L170).